builtin/game/chat.lua

```diff
@@ -666,7 +666,9 @@ core.register_chatcommand("mods", {
 	description = S("List mods installed on the server"),
 	privs = {},
 	func = function(name, param)
-		return true, table.concat(core.get_modnames(), ", ")
+		core.show_formspec(name, "modslist",
+			build_mods_formspec())
```

Comment:
Mod formspec might be cached on start.

Reply:
How would do that? Just declare a new varyable with the cache at the beginning of the file?

Reply:
> How would do that? Just declare a new varyable with the cache at the beginning of the file?

Yes, like that.